src/DeviceInterface/initFormSubmissionsApi.js

```diff
@@ -59,6 +58,13 @@ export function initFormSubmissionsApi (forms) {
                     filledForm?.submitHandler('global pointerdown event + filled form')
                 }
             }
+
+            // TODO: Temporary hack to support Google signin in different languages
+            if (/** @type HTMLElement */(event.target)?.closest('#passwordNext button, #identifierNext button')) {
+                // check if there's a form with values
+                const filledForm = [...forms.values()].find(form => form.hasValues())
+                filledForm?.submitHandler('global pointerdown event + google escape hatch')
+            }
```

Comment:
These are specific escape hatches for Google signin forms. Our algo fails when the form is in a different language, so until we fix that larger issue (upcoming) this plugs the whole. There's another such fix below.